crypto/rand/drbg_lib.c

```diff
@@ -229,6 +228,71 @@ int RAND_DRBG_reseed(RAND_DRBG *drbg,
 }
 
 /*
+ * Mix in the specified data without reseeding |drbg|.
+ * 
+ * Used by drbg_add() only.
+ *
+ * Similar to RAND_DRBG_reseed(), but there is no call to 
+ * get_entropy() and the reseed_counter is not reset
```

Comment:
Could you make it more clear that this is about additional data?

Reply:
Agreed, thanks for the suggestion!